core/input/files

```diff
@@ -21,6 +21,7 @@ p_adj_deltacapoffset.cs4r
 p_boundCapCCSindicator.cs4r
 p_boundEmi.cs4r
 p_emiFossilFuelExtr.cs4r
+p_emiFossilFuelExtr2020.cs4r
```

Comment:
are those data already in the current input data?

Reply:
They are in Laurin's latest `/p/projects/rd3mod/inputdata/output_1.27/rev7.22eb-after_2b1450bc_remind.tgz` at least

Reply:
I've been increasing a minor version of `mrremind` every time I create a new file like this, is that the right procedure?